okapi/views/devel/cronreport.php

```diff
@@ -18,7 +18,7 @@ public static function call()
         $response->content_type = "text/plain; charset=utf-8";
         ob_start();
 
-        require_once "okapi/cronjobs.php";
+        require_once __DIR__ . '/../../cronjobs.php';
```

Comment:
Yes because we removed the set include path code. We can remove such includes if we finish for all files the PSR-4 Structure.